datadog/dogshell/wrap.py

```diff
@@ -189,6 +189,8 @@ def main():
     parser.add_option('-b', '--buffer_outs', action='store_true', dest='buffer_outs', default=False,
                       help="displays the stderr and stdout of the command only once it has \
 returned (the command outputs remains buffered in dogwrap meanwhile)")
+    parser.add_option('--tags', action='store', dest='tags', default=False,
+                      help="comma separated list of tags")
```

Comment:
I think setting the `default` to `default=''` would be more consistent.

Nitpicky: can we also set the `type` to `type=string` ?  Even if it's the default value (https://docs.python.org/2/library/optparse.html#optparse.Option.type), I think it makes things clearer :smile: